include/SDL3/SDL_video_capture.h

```diff
@@ -0,0 +1,353 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2022 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+/**
+ *  \file SDL_video_capture.h
+ *
+ *  \brief Video Capture for the SDL library.
+ */
+
+#ifndef SDL_video_capture_h_
+#define SDL_video_capture_h_
+
+#include "SDL3/SDL_video.h"
+
+#include <SDL3/SDL_begin_code.h>
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * The structure used to identify an SDL video capture device
+ */
+struct SDL_VideoCaptureDevice;
+typedef struct SDL_VideoCaptureDevice SDL_VideoCaptureDevice;
+
+#define SDL_VIDEO_CAPTURE_ALLOW_ANY_CHANGE          1
+
+/**
+ *  SDL_VideoCaptureSpec structure
+ *
+ *  Only those field can be 'desired' when configuring the device:
+ *  - format
+ *  - width
+ *  - height
+ *
+ *  \sa SDL_GetVideoCaptureFormat
+ *  \sa SDL_GetVideoCaptureFrameSize
+ *
+ */
+typedef struct SDL_VideoCaptureSpec
+{
+    Uint32 format;          /**< Frame SDL_PixelFormatEnum format */
+    int width;              /**< Frame width */
+    int height;             /**< Frame height */
+} SDL_VideoCaptureSpec;
+
+/**
+ *  SDL Video Capture Status
+ *
+ *  Change states but calling the function in this order:
+ *
+ *  SDL_OpenVideoCapture()
+ *  SDL_SetVideoCaptureSpec()  -> Init
+ *  SDL_StartVideoCapture()    -> Playing
+ *  SDL_StopVideoCapture()     -> Stopped
+ *  SDL_CloseVideoCapture()
+ *
+ */
+typedef enum
+{
+    SDL_VIDEO_CAPTURE_FAIL = -1,    /**< Failed */
+    SDL_VIDEO_CAPTURE_INIT = 0,     /**< Init, spec hasn't been set */
+    SDL_VIDEO_CAPTURE_STOPPED,      /**< Stopped */
+    SDL_VIDEO_CAPTURE_PLAYING      /**< Playing */
+} SDL_VideoCaptureStatus;
+
+/**
+ *  SDL Video Capture Status
+ */
+typedef struct SDL_VideoCaptureFrame
+{
+    Uint64 ticks;               /**< ticks when this was read from the driver */
+    int num_planes;             /**< Number of planes */
+    Uint8 *data[3];             /**< Pointer to data of i-th plane */
+    int row_stride[3];          /**< Row stride of i-th plane */
```

Comment:
We should use the term pitch here, for consistency with SDL_Surface

Reply:
done